system-tests/projects/react-app-webpack-5-unconfigured/README.md

```diff
@@ -1,3 +1,5 @@
+# NOTE: CRA is no longer supported with Cypress 14. This project uses webpack, ejected from create-react-app
+
 # Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I don't understand why we'd leave this.

Reply:
its to test an ejected version of create react app, which is really just a react/webpack application. I'd like to keep it because its almost a system test around a working migration away from cra